gnosis/cowswap
# [Affiliate] Close all affiliate banners after a successful trade

# Summary

Fixes #1707 

Remove info banner when order is recently fulfilled.

  # To Test

1. Navigate through a referral link with a new account.
- You will see `Valid Affiliate code: You can now do your first trade to join the program` banner message
2. Under the new account do the 1st trade in the mainnet.
- Information banner will disappear
